zqd/ingest/log.go

```diff
@@ -48,20 +50,50 @@ func (rw *recWriter) Write(r *zng.Record) error {
 	return nil
 }
 
-func ingestLogs(ctx context.Context, s *space.Space, paths []string, sortLimit int) error {
-	zr, err := scanner.OpenFiles(resolver.NewContext(), paths...)
-	if err != nil {
+// x509.14:00:00-15:00:00.log.gz (open source zeek)
+// x509_20191101_14:00:00-15:00:00+0000.log.gz (corelight)
+const DefaultJSONPathRegexp = `([a-zA-Z0-9_]+)(?:\.|_\d{8}_)\d\d:\d\d:\d\d\-\d\d:\d\d:\d\d(?:[+\-]\d{4})?\.log(?:$|\.gz)`
```

Comment:
```suggestion
var DefaultJSONPathRegexp = regexp.MustCompile(`([a-zA-Z0-9_]+)(?:\.|_\d{8}_)\d\d:\d\d:\d\d\-\d\d:\d\d:\d\d(?:[+\-]\d{4})?\.log(?:$|\.gz)`)
```

Reply:
how come?

Reply:
oh never mind @alfred-landrum - glancing at the change I missed the MustCompile addition, just saw the const->var change. 👍

Reply:
hmm, this change conflicts with the string type of `jsonPathRegexp` https://github.com/brimsec/zq/blob/6abf592dc5ea792f67d51948e9913074f1c77df0/cmd/zq/zq.go#L90 which is a string because it is a CLI-settable. I'd rather not add another field to the `main.Command struct`, so have reverted the change.